Issues with syn

Just experienced this same issue with nushell, and they solved the problem by pinning the 'syn' version. Apparently an incompatible version went out. Here is the error, specifically:
```error[E0432]: unresolved import 'syn::export'
  --> /home/danya/.local/share/cargo/registry/src/github.com-1ecc6299db9ec823/enumset_derive-0.5.0/src/lib.rs:10:10
   |
10 | use syn::export::Span;
   |          ^^^^^^ could not find 'export' in 'syn'

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,6 +46,7 @@ strum = "0.20.0"
 strum_macros = "0.20.1"
 libc = "0.2"
 regex = "1"
+syn = "=1.0.57"
 
 [dependencies.cursive]
 version = "0.15"

